### ISSUE TITLE
Readme correct parse relative links

### ISSUE THREAD

**Issue by pilot** [KnpLabs/KnpBundles#279]
After parsing a readme file time to time we can catch a links inside doc's which is relative and follow to a links at the github.com, for instance http://knpbundles.com/avalanche123/AvalancheImagineBundle there are a list of relative links which follow now to the 404 page

_link to the imagine lib_
![](http://i.imgur.com/EXirp.png)

**Comment by wysow** [comment]
@KnpLabs/knp-team Is someone thinking we should handle this in another way or does it sounds OK for you??

**Comment by docteurklein** [comment]
imho we should handle relative links. we render a gh page, it's our role to generate links correctly.

**Comment by wysow** [comment]
@docteurklein yes in fact you're maybe right, will try to take a look on this ASAP

**Comment by akovalyov** [comment]
Agree with Florian. We should handle exceptions correctly. Also note, that adding the bundle should be as fast as possible to the end user - any requirement from our side will reduce the number of bundles and not everybody will follow it.
